<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="video_path" default="" />
  <arg name="topic_name" default="camera/image_raw" />
  <node name="video_pub_node" pkg="video_pub_py" exec="video_pub_node"> <!--  respawn="true" -->
    <param name="video_path" value="$(var video_path)"/>
    <param name="topic_name" value="$(var topic_name)"/>
  </node>
</launch>
